src/api.js

```diff
@@ -99,6 +99,10 @@ var api = {
     "FIREBASE_CLOUDLOGGING_URL",
     "https://logging.googleapis.com"
   ),
+  cloudMonitroingOrigin: utils.envOverride(
```

Comment:
is this correct? => cloudMonitroingOrigin

Reply:
Thanks, updated